src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -6929,6 +6936,12 @@ subroutine atm_bdy_set_scalars_work( scalars_driving, scalars_new, &
          end if
 
       end do  ! updates now in temp storage
+      !$acc end parallel
+
+      MPAS_ACC_TIMER_START('atm_init_coupled_diagnostics [ACC_data_xfer]')
+      !$acc exit data copyout(scalars_new)
+      MPAS_ACC_TIMER_STOP('atm_init_coupled_diagnostics [ACC_data_xfer]')
+      !$acc exit data delete(scalars_driving)
```

Comment:
Even though it shouldn't contribute, could you put this `exit data delete` inside the timer?

Reply:
This has been addressed by force-push [982098f](https://github.com/MPAS-Dev/MPAS-Model/commit/982098ffad7242b7f83030ebcbe27efec19ee5c5) to [7d1d937](https://github.com/MPAS-Dev/MPAS-Model/commit/7d1d937a5b311db76885086c4dd2d1616fc1fe0f)